scripts/release

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
Can you please add a one-liner comment explaining the purpose of this script and the setup script.